open_tab_tracker/browsers/Firefox.py

```diff
@@ -42,25 +43,43 @@ def lz4json_decompress_file(file: Path) -> Dict[str, Any]:
 
     @classmethod
     def get_firefox_recovery_file(self):
-        """Returns the first recovery file found for Firefox, or None. Currently only works on macOS."""
+        """Returns the latest recovery file found for Firefox, or None."""
+        firefox_profile_paths = []
         match Platform().get_current_os():
             case OS.MAC:
-                firefox_profiles = (
+                firefox_profile_paths.append(
                     Path.home() / "Library/Application Support/Firefox/Profiles/"
                 )
             case OS.LINUX:
-                firefox_profiles = Path.home() / "snap/firefox/common/.mozilla/firefox/"
+                firefox_profile_paths.extend([
+                    Path.home() / "snap/firefox/common/.mozilla/firefox/",
+                    Path.home() / ".mozilla/firefox/"
+                ])
+            case OS.WINDOWS:
+                firefox_profile_paths.extend([
+                    Path(os.environ["APPDATA"]) / "Mozilla/Firefox/Profiles/",
+                    Path(os.environ["LOCALAPPDATA"]) / "Packages/Mozilla.Firefox/LocalCache/Roaming/Mozilla/Firefox/Profiles/"
+                ])
 
-        firefox_profiles = list(firefox_profiles.glob("*.default*"))
+        firefox_profiles = [
+            profile
+            for path in firefox_profile_paths
+            for profile in path.glob("*.default*")
+        ]
```

Comment:
I find this nested list comp slightly hard to parse -- thoughts on a nested loop instead?

```python
for path in firefox_profile_paths:
    for profile in path.glob("*.default*"):
        firefox_profiles.append(profile)
```